pkg/util/util.go

```diff
@@ -40,6 +40,9 @@ import (
 const (
 	// CharSet defines the alphanumeric set for random string generation
 	CharSet = "0123456789abcdefghijklmnopqrstuvwxyz"
+	// MachineListFormatDeprecationError notifies the user that the old
+	// MachineList format is no longer supported
+	MachineListFormatDeprecationError = "Your MachineList items must include Kind and APIVersion"
```

Comment:
Usually if a name ends with `Err` or `Error` is something assigned to `errors.New()` from `pkg/errors`

Reply:
changed to `MachineListFormatDeprecationMessage`